### PR TITLE
update segaxbd

### PR DESCRIPTION
**Pull request by grant2258**
Updated interrupts and io to mame095 smgp still isint working get credit sounds and a bit of a statuc picture and no unmapped read/writes. I also dumbed down the logging for sonic and this driver will only dumb it down as needed can be enabled buy setting verbose like the original code was. This will need to be looked at some other time.

### CODE REVIEW

**Comment by arcadez2003** [comment]
smgp works without the extra comms code on my xbox driver it still has the CPU4 in the rom table but it seems not to matter
so if your mem mapping is fine for it maybe it's the divider code in the machine.?? or maybe you need to check your using the
correct decrypted roms for the set your using as there are many versions of that game supported.

Anyway i added some stuff needed by Line Of Fire dunno if it'll work though we'll see.

**Comment by grant2258** [comment]
@arcadez cant use the sets in earlier mames as they are encrypted unless we add decryption. It seems something to do with the sound cpu. I was going to get round to loffire  but been messing with dmgp we get a static screen now going to add the nvram handler might  just provide a mircle

**Comment by arcadez2003** [comment]
> @Arcadez cant use the sets in earlier mames as they are encrypted unless we add decryption. It seems something to do with the sound cpu. I was going to get round to loffire but been messing with dmgp we get a static screen now going to add the nvram handler might just provide a mircle

Aye i know that bigman it's just i saw you added the comms and were talking about needing the 4th CPU so i thought
it worth posting that my driver doesn't have that code nor that the game seems to need it anyway it seems your
making progress and knowing how ya have a knack for this im sure everything will be good soon enough :)

**Comment by grant2258** [comment]
yea i notice it wasnt hooked up to anything but added it the sake of it. Strange thing the game crashes with the init. I Ill try move it to machine init is worth a try.
